detectors/aws/ecs/ecs.go

```diff
@@ -0,0 +1,106 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ecs
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"go.opentelemetry.io/otel/label"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/semconv"
+)
+
+const (
+	TypeStr           = "ecs"
+	tmde3EnvVar       = "ECS_CONTAINER_METADATA_URI"
+	tmde4EnvVar       = "ECS_CONTAINER_METADATA_URI_V4"
```

Comment:
nit: I like to avoid non-obvious abbreviations when possible and just use whole works- ex `metadataV3EnvVar`

Reply:
fixed